### PR TITLE
Assignment 3 and 4 done

### PR DESCRIPTION
**Pull request by tinotrifan**

### CODE REVIEW

File: Assignment3/tinotrifan-y160077/Assigment3.txt [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+
+#########################################################################################################################
+#########################################################################################################################
+
+
+5. For each of the properties, except rdf:type, that can be applied to instances of the Politician class, how many
+distinct values do they take in those instances?
+
+SELECT DISTINCT ?property COUNT(?property) AS ?distinctValues
```
Comment (ocorcho) [inline Assignment3/tinotrifan-y160077/Assigment3.txt]: This is not right, since you are counting on the property again, not on the values.

**Comment by ocorcho** [comment]
Assignment 3 and changes needed. Now @dachafra can also check in the meantime assignment 4

File: Assignment3/tinotrifan-y160077/Assigment3.txt (hunk 1 above)
Comment (tinotrifan) [reply Assignment3/tinotrifan-y160077/Assigment3.txt]: fixed

**Comment by ocorcho** [comment]
Assigment 3 ok. Now @dachafra  can check assignment 4.

**Comment by dachafra** [comment]
well done!

**Pull request merged**